spec/acceptance/basic_spec.rb

```diff
@@ -17,4 +18,21 @@ class { 'foreman_scap_client':
   describe package('rubygem-foreman_scap_client') do
     it { is_expected.to be_installed }
   end
+
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<~PUPPET
+      class { 'foreman_scap_client':
+        foreman_repo_rel => 'nightly',
+        server           => 'foreman.example.com',
+        port             => 8443,
+        policies         => [ { id: 1, profile_id: 'default', content_path: '/usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml' } ],
```

Comment:
I might be wrong, but doesn't puppet only support the rocket syntax for hashes?

Reply:
I think it's not relevant in test context: https://github.com/theforeman/puppet-foreman_scap_client/blob/master/spec/classes/init_spec.rb#L10-L14

Reply:
til